python_ta/cfg/graph.py

```diff
@@ -472,6 +462,14 @@ def parse_constraint(self, node: NodeNG) -> Optional[ExprRef]:
         """Parse an Astroid node to a Z3 constraint
         Return the resulting expression
         """
+        try:
```

Comment:
Unlike the `Graph` class, here the `Z3Environment` class is so connected to Z3 that I don't think we need a try-except. Instead just include the imports, and if they fail it's fine for the error to be raised.